### PR TITLE
[OPENJDK-3009] mask passwords from java arguments in logs (UBI8)

### PR DESCRIPTION
**Pull request by jmtd**
https://issues.redhat.com/browse/OPENJDK-3009

Backport of #466

### CODE REVIEW

File: modules/run/artifacts/opt/jboss/container/java/run/run-java.sh [hunk 1]
```diff
@@ -217,23 +217,32 @@ get_classpath() {
...
 }
 
-# Ensure that the running UID has the "jboss" passwd metadata
-# XXX: Maybe we should make this an entrypoint for the image?
-function configure_passwd() {
-  # OPENJDK-533: this file is only writeable if the image uses the
-  # nss_wrapper module. ubi8/openjdk-17 does not.
-  if [ -w "$HOME/passwd" ]; then
-    sed "/^jboss/s/[^:]*/$(id -u)/3" /etc/passwd > "$HOME/passwd"
-  fi
```
Comment (jerboaa) [inline modules/run/artifacts/opt/jboss/container/java/run/run-java.sh]: This is an unrelated change. I'd prefer if we did that separately in case we need to revert.

**Review by jerboaa — COMMENTED** [review]
Could we do the `configure_passwd` removal separately?

File: modules/run/artifacts/opt/jboss/container/java/run/run-java.sh (hunk 1 above)
Comment (jmtd) [reply modules/run/artifacts/opt/jboss/container/java/run/run-java.sh]: We can, sure.

**Review by jerboaa — APPROVED** [review]

**Review by sefroberg — APPROVED** [review]
